cli/borges/command.go

```diff
@@ -0,0 +1,127 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	_ "net/http/pprof"
+
+	"github.com/inconshreveable/log15"
+	"github.com/src-d/borges/metrics"
+)
+
+type ExecutableCommander interface {
+	Commander
+	Execute(args []string) error
+}
+
+type Commander interface {
```

Comment:
Why commander and not just `Command`?

Reply:
I saw in the go-flags library that name too, but yes, `Command` and `ExecutableCommand` looks better. Will change that.